yew-macro/src/html_tree/html_tag/mod.rs

```diff
@@ -6,15 +6,43 @@ use super::HtmlPropSuffix as TagSuffix;
 use super::HtmlTree;
 use crate::{non_capitalized_ascii, Peek, PeekValue};
 use boolinator::Boolinator;
+use lazy_static::lazy_static;
 use proc_macro2::Span;
 use quote::{quote, quote_spanned, ToTokens};
+use std::collections::HashSet;
+use std::iter::FromIterator;
 use syn::buffer::Cursor;
 use syn::parse;
 use syn::parse::{Parse, ParseStream, Result as ParseResult};
 use syn::spanned::Spanned;
 use syn::{Ident, Token};
 use tag_attributes::{ClassesForm, TagAttributes};
 
+lazy_static! {
```

Comment:
I think using a `HashSet` is overkill. Would using a match expression instead not offer better performance?

Reply:
This `HashSet` is only used in the macro resolution compiler phase, it should be fine 👍

Reply:
Would it increase compile times?

Reply:
Potentially yeah, but instantiating a small hashset should be negligible

Reply:
I agree that the `HashSet` isn't a major problem, I just feel like a match arm is a simpler solution.

Reply:
I digged into the assembly and it looks like LLVM is pretty clever at optimizing matching to string literals. I rewrote this as `match`.